storage/badger/operation/common.go

```diff
@@ -196,6 +209,26 @@ func removeByPrefix(prefix []byte) func(*badger.Txn) error {
 	}
 }
 
+func batchRemoveByPrefix(prefix []byte) func(tx *badger.Txn, writeBatch *badger.WriteBatch) error {
```

Comment:
Leo [added a suggestion for goDoc](https://github.com/onflow/flow-go/pull/2535#discussion_r960112736)? The comment was resolved (just unresolved it) but still not documentation (?) 😬

Reply:
Added